validation/courseValidation.js

```diff
@@ -7,8 +7,5 @@ module.exports = Joi.object({
   courseSection: Joi.string().required().messages({
     'any.required': 'courseSection is required'
   }),
-  professorId: Joi.string().required().messages({
```

Comment:
How come professorId was removed? It's still needed in the schema

Reply:
Changed it from requiring professorId in the request body to set the professorId as the id of the user making the request (getting the id from the jwt). This way, you can only create/change/delete courses for yourself.